### PR TITLE
doc mcSTUFFins [BEFORE THE NEXT QUESTCRAFT UPDATE]

### PR DESCRIPTION
**Pull request by swaggio7**
-Fixed some grammar
-Added Immersive Portals and updated the text below the recommended mods
-Updated LDL's description

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,25 +64,25 @@ QuestCraft is mildly slow by itself because minecraft is a very badly optimized
...
-- [Mod Menu](https://modrinth.com/mod/modmenu) and [YetAnotherConfigLib](https://modrinth.com/mod/yacl) (Allows you to see and (with YACL) configure installed mods.)
+- [LambDynamicLights](https://modrinth.com/mod/lambdynamiclights) (Adds dynamic lights lights to Minecraft)
+- [Mod Menu](https://modrinth.com/mod/modmenu) and [YetAnotherConfigLib](https://modrinth.com/mod/yacl) (Allows you to see and (with YACL) configure installed mods)
 - [Memory Leak Fix](https://modrinth.com/mod/memoryleakfix) (Fixes random memory leaks to reduce RAM usage)
 - [More Culling](https://modrinth.com/mod/moreculling) (Culls not just leaves, but loads of other things to improve performance. Installing [More Culling Extra](https://modrinth.com/mod/morecullingextra) will also add more culling features!)
 
-At the moment, Iris and Canvas do not work. Distant Horizons support is planned, but may take a while.                                                   
+At the moment, Iris and Canvas do not work. Distant Horizons and Immersive Portals support is planned, but it may take awhile until we get OpenGL shaders working.
```
Comment (Flopgop) [inline README.md:72]: "a while" is 2 separate words.

**Comment by swaggio7** [comment]
Geez, that was fast.

**Comment by Flopgop** [comment]
> Geez, that was fast.

;p

**Comment by swaggio7** [comment]
Ight, I'm done. Going to play some Wii.

**Comment by swaggio7** [comment]
Is FastAnim coming to QCXR?

**Comment by swaggio7** [comment]
Updated the issue template a little bit.

**Comment by swaggio7** [comment]
I have some saddening news to share. I won't be able to do more commits in the rest of the month as I've been very busy with collage and don't have much free time.
f0b

**Comment by swaggio7** [comment]
To quote my comment from Discord, "So basically all of my work is useless as people use the Discord more than the Github".

**Comment by swaggio7** [comment]
SWAG FOR PRESIDENT
SUPPORT SHAL FALL

**Comment by swaggio7** [comment]
This PR is expected to be merged before the next update, seeing as it will change many things.

**Comment by swaggio7** [comment]
I have decided its useless keeping this PR open, as the other already contains the changes I made, excluding some other stuff.

**Pull request closed without merging**